fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -1775,6 +1855,20 @@ struct TenantManagementWorkload : TestWorkload {
 
 	ACTOR Future<Void> _start(Database cx, TenantManagementWorkload* self) {
 		state double start = now();
+		// Only one client will set the tenantId prefix on the metacluster so fetch that value
+		state Reference<ReadYourWritesTransaction> tr = makeReference<ReadYourWritesTransaction>(cx);
+		loop {
+			try {
+				tr->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+				int64_t tenantIdPrefix =
+				    wait(KeyBackedProperty<int64_t>(tenantIdPrefixKey).getD(tr, Snapshot::False, 0));
+				self->tenantIdPrefix = tenantIdPrefix;
+				break;
+			} catch (Error& e) {
+				wait(tr->onError(e));
+			}
+		}
+		TraceEvent("TenantManagementIdPrefix").detail("Prefix", self->tenantIdPrefix);
```

Comment:
We have an existing mechanism (`TestParameters`) for passing data between the clients, I'd recommend using that to use less code.